sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcTestDataSet.java

```diff
@@ -0,0 +1,101 @@
+package org.apache.beam.sdk.io.jdbc;
+
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.postgresql.ds.PGSimpleDataSource;
+
+import javax.sql.DataSource;
+import javax.xml.crypto.Data;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Date;
+
+/**
+ * Manipulates test data used by the JdbcIO tests.
+ *
+ * <p>This is independent from the tests so that for read tests it can be run separately after data store creation rather
+ * than every time (which can be more fragile.)
+ */
+public class JdbcTestDataSet {
+
+  /**
+   * Use this to create the read tables before IT read tests.
+   *
+   * <p>To invoke this class, you can use this command line:
+   * mvn test-compile exec:java -Dexec.mainClass=org.apache.beam.sdk.io.jdbc.JdbcTestDataSet \
+   *   -Dexec.args="--postgresIp=1.1.1.1 --postgresUsername=postgres --postgresDatabaseName=myfancydb \
+   *   --postgresPassword=yourpassword --postgresSsl=false" \
+   *   -Dexec.classpathScope=test
+   * @param args Please pass options from PostgresTestOptions used for connection to postgres as shown above.
+   */
+  public static void main(String[] args) throws SQLException {
+    PGSimpleDataSource dataSource = new PGSimpleDataSource();
+    PipelineOptionsFactory.register(PostgresTestOptions.class);
+    PostgresTestOptions options = PipelineOptionsFactory.fromArgs(args).create().as(PostgresTestOptions.class);
+
+    dataSource.setDatabaseName(options.getPostgresDatabaseName());
```

Comment:
This looks like the same method as in `JdbcIOIT` - can the actual construction of the DataSource be a method in this file that takes a `PipelineOptions` argument?